config/lobby/db/001_create_tables

```diff
@@ -41,7 +41,9 @@ alter table muted_usernames owner to postgres;
 
 create table ta_users (
     username character varying(40) not null primary key,
-    password character varying(60) not null,
+    password character(34) check (char_length(password)=34),
```

Comment:
@RoiEXLab, can you please revert the changes to this file so I can build a test database from this branch?

Reply:
Whoops I must have messed up when resolving merge conflicts... Will fix that